notebooks/ch-algorithms/superdense-coding.ipynb

```diff
@@ -2498,7 +2498,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.9.7"
+   "version": "3.8.5"
```

Comment:
I believe this change isn't related to https://github.com/Qiskit/platypus/issues/715.

Reply:
Totally agree with you. I talked with @frankharkins about these changes before merge it and they seemed to be metadata related with the environment where the notebook was edited so I saw it fine but good point for next time.

Reply:
Oh yeah, the Python version has been downgraded for this notebook @eddybrando